lib/forms/pegasus_form_validation.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'pegasus_form_errors'
 require_relative '../cdo/regexp'
 require_relative '../cdo/geocoder'
+require_relative '../../pegasus/forms/hoc_signup_2017'
```

Comment:
It was necessary for me to add, otherwise I see stacks like the following. Really, I think we need to be requiring all form kinds. In production, somehow these must already be visible from transitive requires?
```
  1) Error:
insert_form#test_0002_creates a forms_geos entry:
NameError: uninitialized constant HocSignup2017
    /Users/asher/code-dot-org/lib/forms/pegasus_form_validation.rb:133:in `const_get'
    /Users/asher/code-dot-org/lib/forms/pegasus_form_validation.rb:133:in `validate_form'
    /Users/asher/code-dot-org/pegasus/helpers/form_helpers.rb:34:in `insert_form'
    test/test_form_helpers.rb:65:in `block (2 levels) in <class:FormHelpersTest>'
```